File: lab/python/textos/inverter_string.py
def inverter(string):
    ultimo_indice = len(string) - 1

    invertido = ""
    for i in range(ultimo_indice, -1, -1):
        invertido += string[i]

    print(invertido)
    return invertido


inverter("casa")

if inverter("radar") == "radar":
    print("É palíndromo")

File: lab/javascript/textos/inverter_string.js
function inverter(string) {
    let invertido = "";
    for (let i = string.length - 1; i >= 0; i--) {
        invertido += string[i];
    }
    console.log(invertido);
    return invertido;
}

inverter("casa");

if (inverter("radar") === "radar") {
    console.log("É palíndromo");
}
